app/src/main/java/org/andstatus/todoagenda/prefs/EventSource.java

```diff
@@ -0,0 +1,32 @@
+package org.andstatus.todoagenda.prefs;
+
+public class EventSource {
```

Comment:
We need all event providers to be enumerated (including the "Calendar events provider"), and EventSource should include a value of this enum for unique identification.
This opens a way to identify "results" (do you remember that now we store them all in one list,,,)
I could do this myself...